### PR TITLE
Cwl tweaks

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Review by joergfunger — APPROVED** [review]
Nice work, thanks for the effort. Just one nice to have feature (that is not required at all).

**Comment by pdiercks** [comment]
Hi @dglaeser , should we merge this?

**Comment by dglaeser** [comment]
hi @pdiercks, from my side this is fine, but the two open threads were initiated by @joergfunger . I guess the second one is just a comment, not sure about the first one, though..

**Comment by joergfunger** [comment]
Can anyone access the [link](https://w3id.org/cwl/view/git/177943beba5afdefed0b3b6e3c62d84404831d6c/simple_use_case/cwl/wf_run_use_case.cwl) in the open remark?

**Comment by pdiercks** [comment]
> Can anyone access the [link](https://w3id.org/cwl/view/git/177943beba5afdefed0b3b6e3c62d84404831d6c/simple_use_case/cwl/wf_run_use_case.cwl) in the open remark?

no, I cannot access that

**Comment by dglaeser** [comment]
That is because the commit SHA does not exist, I think. This works for me: https://view.commonwl.org/workflows/github.com/BAMresearch/NFDI4IngScientificWorkflowRequirements/blob/main/simple_use_case/cwl/wf_run_use_case.cwl

I will use this link for now, and then after we merge this we just substitute the commit sha with the current tip on main, ok?

File: simple_use_case/cwl/README.md [hunk 1]
```diff
@@ -31,3 +31,6 @@ Note that there exist tools to visualize, edit or create cwl workflows. For inst
 visualize workflows contained in git repositories with [view.commonwl.org](https://view.commonwl.org/),
 or you can use the [Rabix Composer](https://github.com/rabix/composer) to compose workflows locally
 on your machine.
+
+Example: ![Workflow Diagram](workflow.svg)
+[source](https://w3id.org/cwl/view/git/177943beba5afdefed0b3b6e3c62d84404831d6c/simple_use_case/cwl/wf_run_use_case.cwl)
```
[the remark this answers is not in the document]
Comment (dglaeser) [reply simple_use_case/cwl/README.md:36]:
```suggestion
[source](https://view.commonwl.org/workflows/github.com/BAMresearch/NFDI4IngScientificWorkflowRequirements/blob/main/simple_use_case/cwl/wf_run_use_case.cwl)
```

**Review by joergfunger — APPROVED** [review]

**Pull request merged**